src/constants.js

```diff
@@ -35,6 +36,7 @@ export const SettingIds = {
 export const LegacySettingIds = {
   ANON_CHAT: 'anonChat',
   AUTO_THEATRE_MODE: 'autoTheatreMode',
+  AUTO_LEAVE_RAIDS: 'autoLeaveRaids',
```

Comment:
There's no need to update the `LegacySettingIds`, this simply contains all the pre-existing settings that weren't stored in the settings dict.